### DISCUSSION TITLE
GraphQL: Unable to query page content with read permissions

### DISCUSSION THREAD

**Discussion by madodig** [requarks/wiki#6696] [Error / Bug Report] [answered]
Hi,

querying page content via GraphQL is denied, unless the page rules are modified to additionally include both write and manage permissions for the page. User is member of a single group and is allowed to read specific page.

Example request for page with ID 1:
`query { pages { single( id: 1 ) { content } } }`

I've tried the request with both API and JWT tokens.

<br>

Additional info:
- missing manage permission results in "You are not authorized to view this page.",
- missing write or read permissions results in "Forbidden".

<br>

Wiki 2.5.300 running as systemd on 
AlmaLinux 8.8 with 
Node.js 18.16.1 and 
PostgreSQL 15.2.

I haven't tried this on older versions.

**Comment by NGPixel** [comment, marked as answer]
Duplicate of #6672

**Comment by madodig** [comment]
Thanks!
